pkg/deleter/deleter.go

```diff
@@ -67,7 +67,7 @@ func NewDeleter(config *common.RuntimeConfig, cleanupTracker *CleanupStatusTrack
 
 // DeletePVs will scan through all the existing PVs that are released, and cleanup and
 // delete them
-func (d *Deleter) DeletePVs() {
+func (d *Deleter) DeletePVs(deleteFn ...func(pv *v1.PersistentVolume) error) {
 	for _, pv := range d.Cache.ListPVs() {
```

Comment:
why is this a variadic argument? Are we expecting more than one callback or this is mainly because - so as we don't have to modify signature of all calls to `DeletePVs` ? If later - my vote is to fix those calls and keep just one function argument.

Reply:
Also some rudimentary unit tests would be nice.

Reply:
There might be other people who import DeletePVs() in external projects

Reply:
(though I doubt it)

Reply:
Will add unit tests